equations.md

```diff
@@ -6,13 +6,20 @@ $$
 deposit = withdrawal
 $$
 
-Swaps between OHM and sOHM \(staking and unstaking\) are always honored 1:1
+Swaps between $OHM and $sOHM during staking and unstaking are always honored
+1:1. The amount of $OHM deposited into the staking contract will always result
+in the same amount of $sOHM. And the amount of $sOHM withdrawn from the staking
+contract will always result in the same amount of $OHM.
```

Comment:
```suggestion
Swaps between OHM and sOHM during staking and unstaking are always honored
1:1. The amount of OHM deposited into the staking contract will always result
in the same amount of sOHM. And the amount of sOHM withdrawn from the staking
contract will always result in the same amount of OHM.
```

Reply:
I removed the cash tag as the medium post does not use cash tag when referring to tokens such as OHM and DAI.